addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/CookieSameSiteScanRule.java

```diff
@@ -62,19 +66,30 @@ private void checkCookies(HttpMessage msg, int id, String cookieHeader) {
         if (cookies.isEmpty()) {
             return;
         }
+
+        Set<String> ignoreList = CookieUtils.getCookieIgnoreList(getModel());
+
         for (String cookie : cookies) {
             if (CookieUtils.isExpired(cookie)) {
                 continue;
             }
+            boolean isInIgnoreList = ignoreList.contains(CookieUtils.getCookieName(cookie));
```

Comment:
This should be done in the above if, same reason as the previous comment.

Reply:
Updated PR